Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -962,6 +970,15 @@ void vtkSlicerVolumeRenderingLogic::UpdateDisplayNodeFromVolumeNode(
     displayNode->SetAndObserveVolumePropertyNodeID(propNode->GetID());
     }
 
+  // Create ShaderProperty node if needed
+  if( displayNode->GetShaderPropertyNode() == nullptr )
+  {
```

Comment:
Please fix curly brace indentation like this

Reply:
I'm confused about the coding convention. According to [this](https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide#Code_style), Slicer should follow the same convention as VTK for all classes that inherit from VTK classes, which is the case for vtkSlicerVolumeRenderingLogic.

Reply:
Yes, it used to be the same, but VTK changed not long ago. Slicer will follow, but as it basically affects all files when we do it in one go, we want to wait until some really big PRs are in.

Reply:
Ok, I've updated the code and will push in a few minutes